tests/test_rooms/test_query_rooms.py

```diff
@@ -13,6 +13,16 @@
     room_query_with_non_existant_id_response,
     all_remote_rooms_query
 )
+from api.room.schema_query import CalendarApi
+# from mock_data.remote_rooms import remote_rooms
+from helpers.calendar.credentials import (
```

Comment:
Please remember to remove this commented out line. Otherwise good work on working on feedback @verenceLola

Reply:
Done. Thanks @nakatuddesuzan